src/services/ajv.ts

```diff
@@ -17,10 +17,7 @@ export const defaultAjvSettings: any = {
     validateFormats: false,
 };
 
-// eslint-disable-next-line func-names
-export const setDefaultsValidator = (function () {
-    const ajv = createAjv(defaultAjvSettings);
-
+export const addKeywords = (ajv: any) => {
```

Comment:
Minor refactoring so eventually we could create custom AJV instances more easily.